src/plugins/plan/pluginSpec.js

```diff
@@ -48,16 +56,11 @@ describe('the plugin', function () {
         child.style.width = '640px';
         child.style.height = '480px';
         element.appendChild(child);
-
-        openmct.time.timeSystem('utc', {
-            start: 1597160002854,
-            end: 1597181232854
-        });
         openmct.on('start', done);
         openmct.start(appHolder);
     });
 
-    afterEach(() => {
+    afterAll(() => {
```

Comment:
It's best practice with testing for every test spec to start in the same state. This should stay in `afterEach`.